Automattic/jetpack
# Top Posts Widget: check for user privileges when we have that info

Fixes #23400

#### Changes proposed in this Pull Request:

* We were already trying to check for an admin user, but were not doing it in the right spot.

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a brand new site, connect to WordPress.com.
* In Jetpack > Settings, ensure that both the Stats module and the extra widgets module is active.
* Go to Apperance > Themes and switch to a theme that supports widgets, such as Twenty Twenty.
* Go to Apperance > Widgets and add a Top Posts widget.
* Go to the frontend:
    * When logged in, you should see a list of recently published posts.
    * When logged out, you should see nothing.
